.circleci/config.yml

```diff
@@ -77,6 +77,58 @@ commands:
       - run:
           name: Test (no default features)
           command: cargo test --all --no-default-features --verbose
+  dependency-checks:
+    steps:
+      - checkout
+      - run:
+          name: Check for security vulnerabilities in dependencies
+          command: |
+            cargo install cargo-audit
+            cargo audit
+      - run:
+          name: Check licence compatibility of dependencies
+          command: |
+            cargo install cargo-license
+            IFS=$'\n'
+            UNACCEPTABLE=
+            # Ref https://www.mozilla.org/en-US/MPL/license-policy/#Licenses_Compatible_with_the_MPL
+            for LINE in `cargo license -t | tail -n +2`; do
```

Comment:
I'm not thrilled about all this inline bash, if we like the concept maybe we can redo it as a rust program or something.

Reply:
How about a python script? Or even just a bash script really